.vscode/extensions.json

```
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
```

```suggestion
    "EditorConfig.EditorConfig",
```
i think it's case insensitive but ... you never know

---

I copied the idea from the store again and it was lowercase again. Where did you found the PascalCase version?

---

from https://open-vsx.org/extension/editorconfig/editorconfig - but that likely could be a issue in there repo - I'll also submit a pull and will close this

---

https://github.com/open-vsx/publish-extensions/pull/526